src/asciilogo.h

```diff
@@ -9,8 +9,8 @@
 const char *ascii_logo =
 "                _._                                                  \n"
 "           _.-``__ ''-._                                             \n"
-"      _.-``    `.  `_.  ''-._           Redis %s (%s/%d) %s bit\n"
-"  .-`` .-```.  ```\\/    _.,_ ''-._                                  \n"
+"      _.-``    `.  `_.  ''-._           Redis Community Edition      \n"
+"  .-`` .-```.  ```\\/    _.,_ ''-._     %s (%s/%d) %s bit\n"
```

Comment:
just to make it clear, the first `%` is "standalone" / "cluster" / "sentinel".
so in the past it would say "Redis cluster", and now it'll say "Redis Community Edition" in one one, and just "cluster" in the other.
i don't see a problem with that, but wanted to make sure it's clear to everyone.

Reply:
Actually it is the version.